src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    if (inverse.hasOwnProperty([robot[key]])) {
+      return null;
+    } else {
+      inverse[robot[key]] = key;
+    }
```

Comment:
p3 - https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md

It's not this task's checklist, but still, this advice is important